lib/compilers/jakt.ts

```diff
@@ -49,8 +50,8 @@ export class JaktCompiler extends BaseCompiler {
         maxSize: number,
         intelAsm,
```

Comment:
aren't these also `boolean` too?

Reply:
Not sure, this matches the definition in BaseCompiler at the moment

Reply:
got it :)